doc/content/bib/articles_using_salamander.bib

```diff
@@ -8,3 +8,10 @@ @article{Franklin2025
     doi = {10.1016/j.fusengdes.2025.115128},
     author = {Trevor Franklin and Casey Icenhour and Pierre-Clément A. Simon and Paul Humrickhouse and Fande Kong and Lane B. Carasik}
 }
+
+@article{eltawila2025,
+    title = {Thermomechanics Coupling to Monte Carlo Particle Transport on Unstructured Mesh Geometries},
+    year = {2025},
```

Comment:
```suggestion
    year = {2025},
    journal = {Journal TBD, Preprint on \url{https://ssrn.com/abstract=5379499}},
```
make sure to build the documentation locally before pushing to fix these issues. 
so that the documentation failure stops.